### ISSUE TITLE
the problem of loss 

### ISSUE THREAD

**Issue by destiny311** [xuannianz/keras-fcos#2]
hello, the loss (centerness loss)  of the paper,you aren't used in the code of loss,right?

**Comment by xuannianz** [comment]
Hi, the centerness loss is specified here.

https://github.com/xuannianz/keras-fcos/blob/ee6546d9907ce829388097a4fa359e2d2a3f2045/train.py#L121

**Comment by destiny311** [comment]
在论文中,centerness∗=√min(l∗,r∗)max(l∗,r∗)×min(t∗,b∗)max(t∗,b∗) ,但在代码中是否有体现出来呢

**Comment by xuannianz** [comment]
有的
https://github.com/xuannianz/keras-fcos/blob/ee6546d9907ce829388097a4fa359e2d2a3f2045/generators/generator.py#L496
